workflows/scRNAseq/pseudobulk-worflow-decoupler-edger/pseudo-bulk_edgeR-tests.yml

```diff
@@ -0,0 +1,54 @@
+- doc: Test outline for pseudo-bulk_edgeR
+  job:
+    Source AnnData file:
+      class: File
+      location: https://zenodo.org/records/13929549/files/Source%20AnnData%20file.h5ad?download=1
+      filetype: h5ad
+    contrasts_files:
+      class: File
+      path: test-data/contrasts_files.txt
+      filetype: txt
+    'Pseudo-bulk: Fields to merge': null
+    Groupy column: cell_type
+    Sample key column: individual
+    Name Your Raw Counts Layer: counts
+    Factor fields: disease
+    Gene symbol column: gene_symbol
+    Formula: '~ 0 + disease '
+  outputs:
+    'pseudobulk_count_matrix':
```

Comment:
Ca you give the outputs human readable labels, i.e. no underscores ?

Reply:
Hi Marius! Thanks a lot for your feedback. I get this error when trying to give a more human readable name, I think it is not possible. Is there a way to do it or should I just keep it as it was? Thanks a lot!  .. ERROR: Test found for unknown workflow output [Pseudobulk Count Matrix], workflow outputs [['_anonymous_output_1', '_anonymous_output_2', '_anonymous_output_3', '_anonymous_output_4', '_anonymous_output_5', '_anonymous_output_6', '_anonymous_output_7', 'pseudobulk_count_matrix', 'Pseudobulk Plot', 'Filtered by expression', 'Tables: DEG ', 'Report Results: HTML File', 'Tables for volcano plot', 'Volcano Plot on input dataset(s): PDF']]
.. ERROR: Test found for unknown workflow output [Tables for Volcano Plot], workflow outputs [['_anonymous_output_1', '_anonymous_output_2', '_anonymous_output_3', '_anonymous_output_4', '_anonymous_output_5', '_anonymous_output_6', '_anonymous_output_7', 'pseudobulk_count_matrix', 'Pseudobulk Plot', 'Filtered by expression', 'Tables: DEG ', 'Report Results: HTML File', 'Tables for volcano plot', 'Volcano Plot on input dataset(s): PDF']]

Reply:
You have to update the labels in the workflow as well.